### PR TITLE
We're not properly setting the committer & author information in commits.

### PR DESCRIPTION
**Pull request by smn**
See the `From: <>` in https://github.com/universalcore/unicore-cms-content-mama-gt-prod/commit/6f6c8b634ad7392396d9148ac1dab4086fe130b2.patch

### CODE REVIEW

**Comment by smn** [comment]
@miltontony & @Rizziepit ready for review.
Probably should add a test.

File: cms/models.py [hunk 1]
```diff
@@ -486,6 +487,8 @@ def auto_save_post_to_git(sender, instance, created, **kwargs):
     if created:
         Post.objects.exclude(pk=instance.pk).update(position=F('position') + 1)
 
+    print 'get_author_info(instance.last_author)', get_author_info(instance.last_author)
```
Comment (miltontony) [inline cms/models.py]: Stray print

**Comment by Rizziepit** [comment]
Looks good to me

**Comment by miltontony** [comment]
:+1:

**Pull request merged**